Add Project Loved: June 2021 news post (#5707)
ppy/osu-wiki: news/2021-06-15-project-loved-june-2021.md

[![](/wiki/shared/news/2021-06-15-project-loved-june-2021/1449950.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})

[Various Artists - Nakano Yuko Map Collection](https://osu.ppy.sh/beatmapsets/1449950#mania) by [Nakano Yuko](https://osu.ppy.sh/users/8856244)\
200 BPM, 7:02 | 4.90★ – 12.49★\
The \[(Marathon)\], \[Penoreri - Lancelot - Flame of the Rebellion -\] and \[Camellia - Abyss of Frobenoid\] difficulties are *not* being nominated for Loved.\
*description written by [Alter-](https://osu.ppy.sh/users/4980256)*

> Nakano Yuko, a prominent osu!mania mapper bringing forth many expressive charts for 7K. Uploading majority of his maps under -Melon, you'll be able to find a wide range of difficult technically challenging charts paired with various artists most popular songs being used such as songs from Camellia's "Blackmagik Blazing" Album, along with Memme, xi, Lapix and more! Alongside that in this 7K collection compiled of Nakano Yuko's most famous charts, you'll find very straightforward and fun BMS charting styles in the lower spectrum and eventually gaining more technicality in speed and stamina.

Reviewer: ```suggestion
Nakano Yuko is prominent osu!mania mapper bringing forth many expressive charts for 7K. Uploading majority of his maps under the alias -Melon, searching would reward you with a wide range of difficult technically challenging charts paired with various artists most popular songs being used such as songs from Camellia's "Blackmagik Blazing" Album, along with Memme, xi, Lapix and more! 

Alongside that in this 7K collection compiled of Nakano Yuko's most famous charts, you'll find very straightforward and fun BMS charting styles in the lower spectrum and eventually gaining more technicality in speed and stamina.
```
